extensions/visualizations/OppiaVisualizationEnumeratedFrequencyTableDirective.js

```diff
@@ -13,7 +13,8 @@
 // limitations under the License.
 
 /**
- * @fileoverview Directives for all reusable data visualization components.
+ * @fileoverview Directive for all visualization enumerted frequency
```

Comment:
I think this fileoverview is incorrect. Why "all"? And enumerated is spelled wrong.

Reply:
done